omniduct/caches/_serializers.py

```diff
@@ -0,0 +1,25 @@
+import pickle
+
+
+class Serializer(object):
+
+    def file_extension(self):
```

Comment:
@property ?

Reply:
I originally had them as properties, then changed changed them because I wanted to use some other classes but couldn't instantiate them without arguments (I needed to use these methods as classmethods). I then changed my mind about that, so I think we can go back to having them as properties. Thanks for pointing it out!